services/pkgreview/package-rating.service.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  name: Joi.string().required(),
+  type: Joi.string().required(),
+  icon: Joi.string().required(),
+  starString: [Joi.string().optional(), Joi.allow(null)],
+  reviewsCount: Joi.number(),
+  rating: [Joi.number().optional(), Joi.allow(null)],
+}).required()
+
+module.exports = class PkgreviewRating extends BaseJsonService {
+  static get category() {
+    return 'rating'
+  }
+
+  static get route() {
+    return {
+      base: 'pkgreview/rating',
+      pattern: ':pkgManager/:pkgSlug+',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'pkgreview.dev Package Ratings',
+        namedParams: { pkgManager: 'npm', pkgSlug: 'react' },
+        staticPreview: {
```

Comment:
You can make a call to the `render` function here instead of specifying individual keys

Reply:
Makes sense!